### ISSUE TITLE
Update README

### ISSUE THREAD

**Issue by xxxcrel** [moquette-io/moquette#597]
please update README.md, i can't execute gradlew script

**Comment by ashkanxy** [comment]
I think gradlew is short form of gradle wrapper command, if that is true you may need to [install](https://docs.gradle.org/current/userguide/installation.html#installation) it and update the environmental variable to run, but i However, I still could not run build it either by using "gradle wrapper" or "mvn clean package" as discussed in this [doc](https://moquette-io.github.io/moquette/documentation.html#_embedding_into_other_projects).

**Comment by hylkevds** [comment]
Yep, should be updated to `mvnw`, since the project now uses Maven, not Gradle.

**Comment by xxxcrel** [comment]
> Yep, should be updated to `mvnw`, since the project now uses Maven, not Gradle.

I mean should provide maven related command

**Comment by ashkanxy** [comment]
I have installed mvn wrapper following this [repo](https://github.com/takari/takari-maven-plugin)
Then tried `./mvnw clean package`.
But i get the same error that i reported in a separated thread ([#605](https://github.com/moquette-io/moquette/issues/605)).

**Comment by landonmax0** [comment]
please update a more detailed README.md, especially about how to build and package. we can NOT run  moquette.bat

**Comment by cfasana** [comment]
For those who may still be interested in building Moquette, I was able to do it following these instructions:

- Clone the repository: `git clone https://github.com/moquette-io/moquette.git`
- Navigate to the folder of the cloned repository: `cd moquette`
- Switch to the directory of the latest release: `git checkout tags/v0.17.0-RC1`
- Use Maven to build the project: `./mvnw clean install`

If everything works properly you should see `BUILD SUCCESS` at the end of the build process.
If any test doesn't work, try to understand the reason behind the failure. As an example, my tests initially failed since port 1883 was already used.
Alternatively, run `./mvnw clean install -DskipTests` to avoid running tests.
